relay-wstring/src/lib.rs

```diff
@@ -380,10 +408,10 @@ impl fmt::Display for WStr {
 /// These surrogate code units have the first 6 bits set to a fixed prefix identifying
 /// whether they are the *leading* or *trailing* code unit of the surrogate pair.  And for
 /// the leading surrogate this bit prefix is `110110`, thus all leading surrogates have a
-/// code unit between 0xD800-0xDBFF.
+/// code unit of `0xD8xx`.
```

Comment:
I think this wording should not have been changed.  The full range here goes higher than 0xD8FF which is the highest suggested here.